xonsh/parsers/v38.py

```diff
@@ -365,7 +365,7 @@ def p_varargslist_v10(self, p):
             kwarg=p[9],
             defaults=[],
         )
-        self._set_regular_args(p0, p[1], p[2], p[3], p[4])
+        self._set_regular_args(p0, *p[1:5:1])
```

Comment:
I think the step in the slice is not necessary.

Reply:
Yes, reasonable. I remove the step in the new commit.